extras/nginx/locations/tropo.conf.j2

```diff
@@ -1,5 +1,27 @@
 location /assets {
+    # Production Mode: By default,
+    # Troposphere serves static files by saving
+    # necessary staticfiles and webpack output to:
+    #     `{{ASSETS_PATH}}`
+{% if ENABLE_WEBPACK_DEV_SERVER == True %}
+    # alias {{ ASSETS_PATH }};
+{%- else %}
     alias {{ ASSETS_PATH }};
+{%- endif %}
+
+    # Developer mode: For easy development on the GUI
+    # Webpack-dev-server can also be used to host assets
+    # and refresh the clients page as content is updated
+    # To enable webpack-dev-server:
+    # - Ensure that the output of webpack-dev-server includes the line:
+    #       `Project is running at https://{{SERVER_URL}}:8080/`
+    # - uncomment the 'proxy_pass' line below.
+    # - comment out the 'alias' line above
+{% if ENABLE_WEBPACK_DEV_SERVER == True %}
```

Comment:
Okay, this would be the first comment in `variables.ini.dist` that is _not_ inline,

Additionally, the content in the readme won't make as much sense in `variables.ini` as it does inside the template.. Going to post another commit with a "compromise solution" 😄